locust/test/test_main.py

```diff
@@ -1405,7 +1720,7 @@ def on_test_stop(environment, **kwargs) -> None:
                 class LoadTestShape(LoadTestShape):
                     def tick(self):
                         run_time = self.get_run_time()
-                        if run_time < 2:
+                        if run_time < 10:
```

Comment:
This changes the test a little bit. Before, it was actually testing that the test exits, because tick() eventually returns None, now we kill it before that has a chance to happen.